src/recipe/file.service.ts

```diff
@@ -10,24 +9,15 @@ export class FileService {
     private s3Service: S3Service,
   ) {}
 
-  generateS3Key(fileName: string) {
-    const ex = fileName.split('.')[1];
-    return `${randomUUID()}.${ex}`;
-  }
-
-  async uploadFileToS3(file: Express.Multer.File, key: string) {
+  async uploadFileToS3(file: Express.Multer.File): Promise<string> {
     const bucket =
```

Comment:
I think this could be done inside s3Service

Reply:
so what do i need fileService then